host-bin/enter-chroot

```diff
@@ -436,9 +445,22 @@ elif [ ! -f "$shares" ]; then
 downloads ~/Downloads
 EOF
 fi
+
+activeuser=$(getactiveuser)
```

Comment:
I don't think this needs to be a separate function

Reply:
Ok... Since we're not creating /etc/crouton/user anymore (see next comment), there's no reason for it.